github/GithubObject.py

```diff
@@ -302,6 +302,13 @@ def last_modified(self) -> Optional[str]:
         """
         return self._headers.get(Consts.RES_LAST_MODIFIED)  # type: ignore
 
+    @property
+    def last_modified_datetime(self) -> Optional[datetime]:
+        """
+        :type: datetime
+        """
+        return self._makeDatetimeAttribute(self._headers.get(Consts.RES_LAST_MODIFIED))  # type: ignore
```

Comment:
We can simplify this to
```suggestion
        return self._makeDatetimeAttribute(self.last_modified)  # type: ignore
```

Reply:
Method `self._makeDatetimeAttribute` returns an `Attribute`, but we want to return the value, so we need to
```suggestion
        return self._makeDatetimeAttribute(self._headers.get(Consts.RES_LAST_MODIFIED)).value  # type: ignore
```